### PR TITLE
perfscale data-path test requires extra ports to be allowed in gcloud

### PR DESCRIPTION
**Pull request by mukrishn**
OSD (managed GCP) cluster needs some post install steps to allow few range of ports for data-path tests especially k8s-netperf to work.

Include this step in to all jobs that runs data-path test on OSD-GCP.

### CODE REVIEW

**Comment by mukrishn** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-nightly-x86-perfscale-gcp-data-path-6nodes-periodic

**Comment by svetsa-rh** [comment]
@mukrishn So this is for OSD GCP, not Self managed GCP? I didn't even realize that we have OSD-GCP until now :(

File: ci-operator/step-registry/openshift-qe/perfscale-gcp-data-path-fw/openshift-qe-perfscale-gcp-data-path-fw-commands.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+if ! gcloud auth list | grep -E "\*\s+${sa_email}"
+then
+  gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
+  gcloud config set project "${GOOGLE_PROJECT_ID}"
+fi
+
+REGION="${LEASED_RESOURCE}"
+echo "Using region: ${REGION}"
+
+CLUSTER_NAME=$(oc get infrastructure cluster -o json | jq -r '.status.apiServerURL' | awk -F.  '{print$2}')
```
Comment (svetsa-rh) [inline ci-operator/step-registry/openshift-qe/perfscale-gcp-data-path-fw/openshift-qe-perfscale-gcp-data-path-fw-commands.sh:23]: Related discussion https://github.com/openshift/release/pull/44173#discussion_r1355334926

**Comment by mukrishn** [comment]
going thru, should be good once finished 
```
INFO[2023-10-11T17:12:48Z] Running step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-perfscale-gcp-data-path-fw. 
INFO[2023-10-11T17:14:10Z] Step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-perfscale-gcp-data-path-fw succeeded after 1m22s. 
```

File: ci-operator/step-registry/openshift-qe/perfscale-gcp-data-path-fw/openshift-qe-perfscale-gcp-data-path-fw-commands.sh (hunk 1 above)
Comment (mukrishn) [reply ci-operator/step-registry/openshift-qe/perfscale-gcp-data-path-fw/openshift-qe-perfscale-gcp-data-path-fw-commands.sh:23]: just thru, we have dittybopper jobs before this so it is expected api server should be available at this stage.
```
INFO[2023-10-11T17:12:48Z] Running step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-perfscale-gcp-data-path-fw. 
INFO[2023-10-11T17:14:10Z] Step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-perfscale-gcp-data-path-fw succeeded after 1m22s. 
```

Comment (svetsa-rh) [reply ci-operator/step-registry/openshift-qe/perfscale-gcp-data-path-fw/openshift-qe-perfscale-gcp-data-path-fw-commands.sh:23]: Shouldn't block us but as we keep learning from the related issue, we may need to revisit how we are retrieving CLUSTER_NAME if needed.

**Comment by mukrishn** [comment]
Passed!
```
INFO[2023-10-11T17:12:48Z] Running step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-perfscale-gcp-data-path-fw. 
INFO[2023-10-11T17:14:10Z] Step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-perfscale-gcp-data-path-fw succeeded after 1m22s. 
INFO[2023-10-11T17:14:10Z] Running step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-network-smoke. 
INFO[2023-10-11T17:19:32Z] Step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-network-smoke succeeded after 5m21s. 
INFO[2023-10-11T17:19:32Z] Running step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-network-perf. 
INFO[2023-10-11T18:51:52Z] Step perfscale-gcp-data-path-6nodes-periodic-XXXXXXXXXXXX-network-perf succeeded after 1h32m20s. 
```

**Pull request merged**